src/app-sync/services/files-service.js

```diff
@@ -110,14 +116,53 @@ class FilesService {
         file.encrypt_test,
         file.encrypt_keyid,
         deletedInt,
+        file.owner,
       ],
     );
   }
 
-  find(limit = 1000) {
-    return this.accountDb
-      .all('SELECT * FROM files WHERE deleted = 0 LIMIT ?', [limit])
-      .map(this.validate);
+  find({ userId = null, limit = 1000 } = {}) {
+    const canSeeAll = userId === null || isAdmin(userId);
```

Comment:
This code as written seems a little dangerous to me. In particular, it's used in `app-sync` which just passes in `res.locals.user_id`—right now, it may be the case that this is always non-null, but I can imagine a future PR which accidentally makes that not the case. Do we need the `null` fallback to admin for something? If not, can we remove it and error loudly if the user ID is null?

Reply:
Its nullable because it was messing with the test cases. Fixed the test cases and removed the null